src/cli/cli.cpp

```diff
@@ -2091,6 +2091,33 @@ otError Interpreter::ProcessIpAddr(Arg aArgs[])
         {
             OutputIp6AddressLine(*otThreadGetMeshLocalEid(GetInstancePtr()));
         }
+        else if (aArgs[0] == "printorigin")
```

Comment:
Can we use argument like `-v` to have more verbose output ?

Reply:
Thanks for your fast code review! Already changed the input argument to `-v`.